### ISSUE TITLE
Lag/performance issues

### ISSUE THREAD

**Issue by Flexo013** [Flexo013/HeadsWebsite#7]
Some friends of mine have reported that the website is quite heavy on their laptops and not-so-powerful computers. Even the light collection is on the heavy side. (There is no light collection for the main collection.) Maybe we could add some kind of paging to the light collection, so that we only display 100 heads per page.

**Comment by Flexo013** [comment]
I like the addition of the paging! That should really help users out. Maybe we can give the user a choice for the amount of heads they want to display per page? (I would say 10, 25, 50, and 100 as the options.)

Currently the spacing on the Light Collection seems somewhat messed up (in Google Chrome):
![screenshot083](https://cloud.githubusercontent.com/assets/4507015/4927734/37d7c4d4-653e-11e4-958c-a022f0daefb9.png)

**Comment by Fancypants** [comment]
I consider this more as an enhancement then a bug but I'm not that picky. I added a pagination which only shows 10 results (100 would still be a lot and you'd have to scroll further down to view the pagination, might change it if we believe 10 is not enough). I'll see what I can do about the heads per page.

P.S. You took a screenshot while I was testing stuff ;). Looks normal now.

**Comment by Flexo013** [comment]
Looks great now indeed! I changed the labels. (You can do this next time as well ;) )

**Comment by Fancypants** [comment]
I noticed the User Collection (non-light version) was getting pretty big too so I decided to split these heads up so it would show less heads. I didn't like the idea of pagination since we already use that in the light version (people can jump from the front to the end in light version). I want the User Collection to be different and mostly show the latest heads. I added a function that loads more heads when users scroll down. Check it out! [LINK](http://heads.freshcoal.com/heads.php)
